CHANGELOG.md

```diff
@@ -55,6 +55,7 @@
 - `jaeger` receiver: Escape user input used in output (#2815)
 - `prometheus` exporter: Ensure same time is used for updated time (#2745)
 - `prometheusremotewrite` exporter: Close HTTP response body (#2875)
+- Fix Jaeger receiver to honor TLS Settings (#2866)
```

Comment:
Please add this in the "unreleased" section.

Reply:
Done